text/0000-sends-accepts.md

```diff
@@ -0,0 +1,46 @@
+- Start Date: 2015-12-3
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes adding `sends` and `accepts` attributes to the `Category` element
```

Comment:
@smizell I understand that. What I am grappling with is – are we just going to add "everything everywhere" just because format XYZ has it or are we after a **conceptually consistent data model**?